personio-personnel-data-api-oa3.yaml

```diff
@@ -3828,6 +3969,67 @@ components:
         - end_date
         - half_day_start
         - half_day_end
+    CreateAbsencePeriodRequest:
+      title: Create Absence periods **tracked in hours**
+      type: object
+      properties:
+        employee_id:
+          type: integer
+          description: Employee identifier
+        absence_type_id:
+          type: string
+          format: uuid
+          description: Absence type identifier
+        start_date:
+          type: string
+          format: date
+          description: "Absence start date. Format: yyyy-mm-dd"
+          example: '2020-01-31'
+        end_date:
+          type: string
+          format: date
+          description: "Absence end date. Format: yyyy-mm-dd"
+          example: '2020-01-31'
+        full_day:
+          type: boolean
+          example: false
+          description: Whether it's a full day absence or not, only required if the start_date and end_date are the same (one day absence at most)
+        start_time:
+          type: string
+          format: date
+          description: "Absence start time. Format: HH:ii and only required if the start_date and end_date are the same, and full_day is false (less than one day absence)"
```

Comment:
We usually use the term "partial-day absence", at least internally, which I think is more descriptive than "less than one day absence"